### DISCUSSION TITLE
notifications should have an is_read flag?

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13813] [Feature Requests]
To silence the number showing on top of the notifications icon, an is_read flag could be used after a user has read a notification.

edit: Better still, a nullable date_read column

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
